scripts/download

```diff
@@ -26,6 +26,7 @@ if [[ -z "${LOCAL_ARTIFACTS}" ]]; then
         SUFFIX="-${ARCH}"
     fi
     curl -sfL -R -o artifacts/k3s https://github.com/k3s-io/k3s/releases/download/${URI_VERSION}/k3s${SUFFIX}
+    ls artifacts
```

Comment:
is this needed, or just added for debugging?

Reply:
yep. It was removed.